scripts/deploy/main.bicep

```diff
@@ -594,7 +594,7 @@ resource cosmosAccount 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' = if (
   location: location
   kind: 'GlobalDocumentDB'
   properties: {
-    consistencyPolicy: { defaultConsistencyLevel: 'Session' }
+    consistencyPolicy: { defaultConsistencyLevel: 'Eventual' }
```

Comment:
I agree that we should move to eventual consistency... Changing from Strong to Eventual though can create some subtle bugs that are intermittent and depend on the timing of calls. Generally the client is build assuming eventual consistency and handles retries for scenarios where it matters such as a post to create a resource directly followed by a get for the same resource. I'm not sure if we have any cases like that currently.

I'm fine with checking this in if we've done a decent walkthrough of the app features and made sure we don't get any weirdness.

Reply:
Ok, back to `Session`